### DISCUSSION TITLE
Error installing bitgo-cli; No git binary found in $PATH

### DISCUSSION THREAD

**Discussion by BTCert** [community/community#21948] [New to GitHub]
<p>Hi Git community,</p>
<p>I have a question regarding the use of the Bitgo-cli (<a href="https://github.com/BitGo/bitgo-cli" rel="nofollow noopener">https://github.com/BitGo/bitgo-cli</a>). I installed node and bitgo-cli, but when I try to install (using the suggested command</p>
<pre><code>npm install -g bitgo-cli
</code></pre>
<p>I get the following error message:</p>
<p>npm ERR! code ENOGIT</p>
<p>npm ERR! No git binary found in $PATH</p>
<p>npm ERR! Failed using git.</p>
<p>npm ERR! Please check if you have git installed and in your PATH.</p>
<p>Is there anyone that can help me solve this? I am not sure where I can find the required Git and how to install it. Apologies for the stupid question, I am a bit on unfamiliar terrain here… </p>
<p>Thanks!<br>
Egbert </p>

**Comment by hbutau** [comment, marked as answer]
<p>Hello Egbert ,</p>
<p>Judging from the output you are getting, it seems you do not have git installed on your computer. May you kindly provide the platform you are running so that I can assist further that is if you do not have git installed.</p>

**Comment by BTCert** [comment]
<p>Thanks Hbutau, you were right! After installing Git everything worked out.</p>
<p>Cheers!</p>
